pkg/synthetictests/allowedalerts/all.go

```diff
@@ -3,33 +3,33 @@ package allowedalerts
 func AllAlertTests() []AlertTest {
 	return []AlertTest{
 		newAlert("etcd", "etcdMembersDown").pending().neverFail(),
-		newAlert("etcd", "etcdMembersDown").firing(),
+		newAlert("etcd", "etcdMembersDown").info(),
 		newAlert("etcd", "etcdGRPCRequestsSlow").pending().neverFail(),
-		newAlert("etcd", "etcdGRPCRequestsSlow").firing(),
+		newAlert("etcd", "etcdGRPCRequestsSlow").info(),
 		newAlert("etcd", "etcdHighNumberOfFailedGRPCRequests").pending().neverFail(),
-		newAlert("etcd", "etcdHighNumberOfFailedGRPCRequests").firing(),
+		newAlert("etcd", "etcdHighNumberOfFailedGRPCRequests").info(),
 		newAlert("etcd", "etcdMemberCommunicationSlow").pending().neverFail(),
-		newAlert("etcd", "etcdMemberCommunicationSlow").firing(),
+		newAlert("etcd", "etcdMemberCommunicationSlow").info(),
 		newAlert("etcd", "etcdNoLeader").pending().neverFail(),
-		newAlert("etcd", "etcdNoLeader").firing(),
+		newAlert("etcd", "etcdNoLeader").info(),
 		newAlert("etcd", "etcdHighFsyncDurations").pending().neverFail(),
-		newAlert("etcd", "etcdHighFsyncDurations").firing(),
+		newAlert("etcd", "etcdHighFsyncDurations").info(),
 		newAlert("etcd", "etcdHighCommitDurations").pending().neverFail(),
-		newAlert("etcd", "etcdHighCommitDurations").firing(),
+		newAlert("etcd", "etcdHighCommitDurations").info(),
 		newAlert("etcd", "etcdInsufficientMembers").pending().neverFail(),
-		newAlert("etcd", "etcdInsufficientMembers").firing(),
+		newAlert("etcd", "etcdInsufficientMembers").info(),
 		newAlert("etcd", "etcdHighNumberOfLeaderChanges").pending().neverFail(),
-		newAlert("etcd", "etcdHighNumberOfLeaderChanges").firing(),
+		newAlert("etcd", "etcdHighNumberOfLeaderChanges").info(),
 
 		newAlert("kube-apiserver", "KubeAPIErrorBudgetBurn").pending().neverFail(),
-		newAlert("kube-apiserver", "KubeAPIErrorBudgetBurn").firing().neverFail(), // https://bugzilla.redhat.com/show_bug.cgi?id=2039539
+		newAlert("kube-apiserver", "KubeAPIErrorBudgetBurn").info().neverFail(), // https://bugzilla.redhat.com/show_bug.cgi?id=2039539
```

Comment:
The refactoring looks good to me, but I am curious about this line. The `KubeAPIErrorBudgetBurn` alert has two severities either `critical` or `warning` so if I am not wrong, this line doesn't do anything since it allows the info level alert 
that doesn't exist to fire?

Reply:
> The refactoring looks good to me, but I am curious about this line. The `KubeAPIErrorBudgetBurn` alert has two severities either `critical` or `warning` so if I am not wrong, this line doesn't do anything since it allows the info level alert
> that doesn't exist to fire?

`.info` fails when it is at or above info level.  otherwise you can get weird results like passing when the alert is critical.  this line is what is adding the flake to so many tests today.

Reply:
Ah make sense, I didn't understood it correctly in the first place, thank you for clarifying.